tests/unit/components/search/ViewToggles.spec.js

```diff
@@ -28,37 +28,41 @@ const factory = () => mount(ViewToggles, {
 describe('components/search/ViewToggles', () => {
   const views = ['list', 'grid'];
 
-  it('displays a toggle for each view', () => {
-    const wrapper = factory();
+  for (const view of views) {
+    describe(`${view} view`, () => {
+      it('has a toggle', () => {
+        const wrapper = factory();
 
-    for (const view of views) {
-      const viewToggle = wrapper.find(`[data-qa="search ${view} view toggle"]`);
-      viewToggle.exists().should.eq(true);
-    }
-  });
-
-  it('links to search with view parameter set', () => {
-    const wrapper = factory();
+        const viewToggle = wrapper.find(`[data-qa="search ${view} view toggle"]`);
+        viewToggle.exists().should.eq(true);
+      });
 
-    const viewToggleLink = wrapper.find('[data-qa="search list view toggle"] a');
+      // TODO: why does this fail? href in tests is just e.g. "?view=list" without
```

Comment:
Took me all morning to resolve. However, I have found the solution. You just need to pass the `path: '/search'` in the `linkGenRoute` property.  So you would end up with something like this:

```
linkGenRoute: {
    name: 'search',
    path: '/search'
}
```

Reply:
I'm reluctant to do this as it's not needed by the application functionality itself.